volume/volume.go

```diff
@@ -87,6 +88,11 @@ func getDrivers(
 
 func (vdm *VolumeDriverManager) Mount(volumeName, volumeID string, overwriteFs bool, newFsType string) (string, error) {
 	for _, driver := range vdm.Drivers {
+		log.WithFields(log.Fields{
+			"volumeName":  volumeName,
+			"volumeID":    volumeID,
+			"overwriteFs": overwriteFs,
+			"newFsType":   newFsType}).Debugf("%s: Mount()", driver.Name())
```

Comment:
Hi @clintonskitson,

I'm only going to comment on this once for this file, but it's applicable in *numerous* places. The `Debugf` and all the `LOGf` functions in `logrus` exist for drop-in compatibility with `stdlib`'s `log` package only. All places where the format variants are used should be changed. For example, the above line should look like this:

```
		log.WithFields(log.Fields{
			"volumeName":  volumeName,
			"volumeID":    volumeID,
			"overwriteFs": overwriteFs,
			"newFsType":   newFsType,
			"driverName": driver.Name()}).Debu("mounting volume")
```

However, if you'll look inside the providers, I took great pains to log most everything at the provider level as possible as all log statements always include the `providerName` field in conjunction with that provider's constant value denoting the same.